app/locales/en.json

```diff
@@ -7,5 +7,170 @@
+  "Report your health condition": "Report your health condition",
+  "Thank you for your contribution! Use this link if your health condition changes:": "Thank you for your contribution! Use this link if your health condition changes:",
+  "Join the most important crowdsource! Regardless if you're healthy or not, please submit the form below – that is also valuable information! ": "Join the most important crowdsource! Regardless if you're healthy or not, please submit the form below – that is also valuable information! ",
+  "Geography and demography": "Geography and demography",
+  "/healthcondition/": "/healthcondition/",
+  "Your age": "Your age",
+  "Choose your age": "Choose your age",
+  "Biological gender": "Biological gender/Sex",
+  "Female": "Female",
+  "Male": "Male",
+  "Zip code": "Zip code",
+  "Testing and symptoms": "Testing and symptoms",
+  "Have you been in close contact with someone who was tested positive for COVID-19?": "Have you been in close contact with someone who was tested positive for COVID-19?",
+  "Yes": "Yes",
+  "No": "No",
+  "Positive": "Positive",
+  "Negative": "Negative",
+  "Have you been tested for COVID-19?": "Have you been tested for COVID-19?",
+  "What was the result?": "What was the result?",
+  "Waiting for result": "Waiting for result",
+  "Symptoms you have experienced": "Symptoms you have experienced",
+  "Select only the symptoms you have experienced": "Select only the symptoms you have experienced",
+  "Dry cough": "Dry cough",
+  "Mucus cough": "Mucus cough",
+  "Fatigue and exhaustion": "Fatigue and exhaustion",
+  "Fever": "Fever",
+  "Shortness of breath": "Shortness of breath",
+  "Muscle pain": "Muscle pain",
+  "Diarrhea": "Diarrhea",
+  "Headache": "Headache",
+  "Sore throat": "Sore throat",
+  "Loss of sense of smell": "Loss of sense of smell",
+  "Loss of sense of taste": "Loss of sense of taste",
+  "Stuffy or runny nose": "Stuffy or runny nose",
+  "When did the symptoms start?": "When did the symptoms start?",
+  "How will my data be used?": "How will my data be used?",
+  "Your health information is stored anonymously, and can not be used to identify you.": "Your health information is stored anonymously, and can not be used to identify you.",
+  "Our privacy policy can be found here": "Our privacy policy can be found here",
+  "I agree to my being data in accordance with the privacy statement": "I agree to my being data in accordance with the privacy statement",
+  "Submit report": "Submit report",
+  "Statistics": "Statistics",
+  "Download the dataset as CSV": "Download the dataset as CSV",
+  "COVID-19 infected": "COVID-19 infected",
+  "In total <%= numberWithSpaces(totalInfectedPeopleWithSymptoms) %> people have reported that they have tested positive for COVID-19 and experience symptoms.": "In total <%= numberWithSpaces(totalInfectedPeopleWithSymptoms) %> people have reported that they have tested positive for COVID-19 and experience symptoms.",
```

Comment:
Did you check if using `<%= numberWithSpaces(totalInfectedPeopleWithSymptoms) %>`works? Should maybe rather be split up.

Reply:
Yeah I overlooked this. It's fixed for one instance but I'll (or someone else) go over the others and fix them.